### PR TITLE
Test commit

### PR DESCRIPTION
**Pull request by laxmanghavte**

### CODE REVIEW

File: pusher.php [hunk 1]
```diff
@@ -8,6 +8,8 @@
 use Monolog\Logger;
 use Monolog\Handler\StreamHandler;
 
+/* new urban airship */
+
```
Comment (laxmanghavte) [inline pusher.php]: Remove comment

Comment (laxmanghavte) [inline pusher.php]: remove extra line

File: pusher.php [hunk 2]
```diff
@@ -17,5 +19,5 @@
     ->setDeviceTypes(P\all)
     ->send();
 print "<pre>";
-print_r($response);
```
Comment (laxmanghavte) [inline pusher.php:20]: Print r

File: test.php.txt [hunk 3]
```diff
@@ -12,4 +12,20 @@ $response = $airship->push()
 
 print "Push sent!. Push IDs:" . $response.push_ids;
 
-?>
\ No newline at end of file
+?>
+
+<?php
+require_once 'vendor/autoload.php';
```
Comment (laxmanghavte) [inline test.php.txt]: Extra code

File: pusher.php (hunk 1 above)
Comment (Mithesh-k) [inline pusher.php]: Remove external

**Pull request closed without merging**